File: apps/site/snippets/voice/livekit.swift
import LiveKit
import SinuaLiveKit   // packages/ios-livekit

// Joins with your backend's access token and publishes the mic. The agent's state
// comes from its `lk.agent.state` attribute (LiveKit Agents set it).
func liveKitVoice(url: String, token: String) -> LiveKitVoiceSource {
    LiveKitVoiceSource(url: url, token: token)
}

// Already have a Room? Pass it; you keep ownership.
func liveKitVoice(room: Room) -> LiveKitVoiceSource {
    LiveKitVoiceSource(room: room)
}

File: apps/site/snippets/voice/livekit.kt
package snippets.voice

import android.content.Context
import dev.sinua.livekit.LiveKitVoiceSource
import io.livekit.android.room.Room

// Joins with your backend's access token and publishes the mic. The agent's state
// comes from its `lk.agent.state` attribute (LiveKit Agents set it).
fun liveKitVoice(context: Context, url: String, token: String) = LiveKitVoiceSource.owned(context, url, token)

// Already have a Room? Pass it; you keep ownership.
fun liveKitVoice(room: Room) = LiveKitVoiceSource(room)
